chain/syncer.go

```diff
@@ -338,18 +346,30 @@ func (syncer *Syncer) HandleNewTipSet(ctx context.Context, ci *types.ChainInfo,
 		return err
 	}
 
+	syncer.reporter.UpdateStatus(syncingStarted(syncer.clock.Now().Unix()), syncHead(ci.Head), syncHeight(ci.Height), syncTrusted(trusted), syncComplete(false))
+	defer syncer.reporter.UpdateStatus(syncComplete(true))
+
```

Comment:
update that we are trying to sync a new head, when we started, and if we trusts it.
when this method exists we are done syncing the new head, it may or may not be our new validated head.